www/js/diary/diaryHelper.ts

```diff
@@ -77,13 +78,6 @@ export function getBaseModeByKey(
   return BaseModes[key] || BaseModes.UNKNOWN;
 }
 
-export function getBaseModeOfLabeledTrip(trip, labelOptions) {
```

Comment:
It was used in 1 or 2 places but getBaseModeByValue does the same thing

(from 4d48aeebd2b1734e3d08dc5de6726f1faf4499bc):
> \- in diaryHelper, getBaseModeOfLabeledTrip was removed since it was basically just a shortcut to getBaseModeByValue with a trip as param instead of its userInput. All usages replaced with getBaseModeByValue